### PR TITLE
Soarian FHIR R4 MedicationRequest Resource Documentation

### PR DESCRIPTION
**Pull request by VamsiMure**
Description
Soarian FHIR R4 MedicationRequest resource

PR Checklist
----
- [ ] Screenshot(s) of changes attached before changes merged.
MedicationRequest-1
MedicationRequest-2
MedicationRequest-3
MedicationRequest-4
MedicationRequest-5
MedicationRequest-6
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/soarian/r4/clinical/medications/medicationrequest.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+## Terminology Bindings
+
+#### Contained Medication Bindings
+
+<%= terminology_table(:soarian_medication_request, :r4) %>
+
+## Search
+
+Search for MedicationRequest resources that meet the specified query parameters
```
Comment (HRNWriter) [inline content/soarian/r4/clinical/medications/medicationrequest.md:40]: Please add a period (.) to the end of this sentence.

PRODOC-160857.

File: layouts/soarian/r4/clinical/medications.html [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<p>Resources in the Medications category are used to support the immunization and medication processes.</p>
+
+<p>Click <a href="https://hl7.org/fhir/r4/medications-module.html">here</a> to see the detailed categorization and full set of resources that fall into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.</p>
```
Comment (HRNWriter) [inline layouts/soarian/r4/clinical/medications.html:3]: We are not supposed to use the "Click here" construction wit h links anymore. Please change this sentence to 

Select <link> for the detailed categorization and full set of resources that fill into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.

PRODOC-160857.

**Review by HRNWriter — COMMENTED** [review]
Please see my comments.

PRODOC-160857.

File: content/soarian/r4/clinical/medications/medicationrequest.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+## Retrieve by ID
+
+List an individual MedicationRequest resource by its ID:
+
+    GET /MedicationRequest/:id
+
+## Authorization Types
+
+<%= authorization_types(provider: true, patient: true)%>
```
Comment (NeethuKP) [inline content/soarian/r4/clinical/medications/medicationrequest.md:106]: Do we need to support system token as well ?

File: layouts/soarian/r4/clinical/medications.html (hunk 2 above)
Comment (VamsiMure) [reply layouts/soarian/r4/clinical/medications.html:3]: <p><a href="https://hl7.org/fhir/r4/medications-module.html">select</a> for the detailed categorization and full set of resources that fall into this category in the HL7<sup>®</sup> FHIR<sup>®</sup> Standard.</p>. Did you mean like this way where 'select' is clickable which redirects page to https://hl7.org/fhir/r4/medications-module.html or did you mean to say the whole sentence is clickable ?

File: content/soarian/r4/clinical/medications/medicationrequest.md (hunk 1 above)
Comment (VamsiMure) [reply content/soarian/r4/clinical/medications/medicationrequest.md]: Done

File: layouts/soarian/r4/clinical/medications.html (hunk 2 above)
Comment (HRNWriter) [reply layouts/soarian/r4/clinical/medications.html:3]: Apologies, I meant to say "Select https://hl7.org/fhir/r4/medications-module.html for the detailed categorization and full set of resources that fill into this category in the HL7® FHIR® Standard." Make https://hl7.org/fhir/r4/medications-module.html a link.
Thank you!

**Review by NeethuKP — APPROVED** [review]

**Review by HRNWriter — APPROVED** [review]
+1

**Review by slvsridhar — APPROVED** [review]

**Comment by PBKarthik** [comment]
After Publish
image
image
image

**Pull request merged**